socorro/external/es/connection_context.py

```diff
@@ -110,6 +110,20 @@ def get_index_template(self):
         """Return template for index names."""
         return self.config.elasticsearch_index
 
+    def get_index_for_date(self, date):
+        """Return the index name for a given date.
+
+        Index names are generated by filling in the date parameters for the
+        specified date in the index template.
+
+        :arg datetime date: the datetime to use to generate the index template
+
+        :returns: index name as a str
+
+        """
+        template = self.get_index_template()
+        return date.strftime(template)
```

Comment:
A different version of this was in crashstorage. I moved it to here and generalized it because it makes writing tests a lot easier.